### PR TITLE
fix: unbound variable _group in reset/dc-detect-version script (#1283)

### PR DESCRIPTION
**Pull request by lovetodream**
This would fix the reset script. [#1283 –_ed._]

Because the `dc-detect-version` is only used in the reset script, I think it is fine to add the _group and _endgroup declarations directly.

### CODE REVIEW

**Review by chadwhitacre — APPROVED** [review]

**Comment by chadwhitacre** [comment]
ref #1283

**Comment by chadwhitacre** [comment]
Blech #1273. 😖

**Comment by lovetodream** [comment]
> Blech #1273. 😖

I Just noticed that #1273 was closed. Does this mean that this can be merged?

**Comment by chadwhitacre** [comment]
> Does this mean that this can be merged?

Almost. #1273 turned into #1289, which (it turns out) basically dupes #1290, which revives #1251, which we had to revert in #1272 because it effed up CI over in `sentry`. Kind of a tangle right now but I think we are making progress. 😁

Hopefully we can get back here soon ... 🤞

**Comment by chadwhitacre** [comment]
Still getting a garbled relay config. 🤔

https://github.com/getsentry/self-hosted/runs/5009812723?check_suite_focus=true#step:5:121

**Comment by aminvakil** [comment]
As https://github.com/getsentry/self-hosted/pull/1315 has been merged, should this fix also reflect GCB too?

**Comment by chadwhitacre** [comment]
Yeah, we need to rebase/update.

File: install/dc-detect-version.sh [hunk 1]
```diff
@@ -1,3 +1,11 @@
+if [ "${GITHUB_ACTIONS:-}" = "true" ]; then
```
Comment (aminvakil) [inline install/dc-detect-version.sh:1]: > Yeah, we need to rebase/update.

@chadwhitacre I mean is there some environment variable or something like that in GCB which should be checked like here?

Comment (chadwhitacre) [reply install/dc-detect-version.sh:1]: This can stay as-is, only GHA parses the `::` in a special way (right?), other envs can all use the arrow output.

**Review by aminvakil — APPROVED** [review]
LGTM unless squashing commits is not an option and this should be rebased without a merge commit.

**Comment by chadwhitacre** [comment]
Thanks for your patience @lovetodream. 🙏

**Pull request merged**